src/Ledger/RolloverLedger/RolloverLedger.js

```diff
@@ -0,0 +1,108 @@
+import React, { useCallback } from 'react';
+import { FormattedMessage } from 'react-intl';
+import PropTypes from 'prop-types';
+
+import stripesFinalForm from '@folio/stripes/final-form';
+import {
+  Accordion,
+  AccordionSet,
+  AccordionStatus,
+  Col,
+  Pane,
+  Paneset,
+  Row,
+} from '@folio/stripes/components';
+import {
+  FormFooter,
+} from '@folio/stripes-acq-components';
+
+import RolloverLedgerBudgets from './RolloverLedgerBudgets';
+
+export const ROLLOVER_LEDGER_ACCORDION = {
+  budgets: 'budgets',
+  encumbrances: 'encumbrances',
+};
+
+export const ROLLOVER_LEDGER_ACCORDION_LABELS = {
+  [ROLLOVER_LEDGER_ACCORDION.budgets]: <FormattedMessage id="ui-finance.ledger.rollover.budgets" />,
+  [ROLLOVER_LEDGER_ACCORDION.encumbrances]: <FormattedMessage id="ui-finance.ledger.rollover.encumbrances" />,
+};
+
+const RolloverLedger = ({
+  goToCreateFY,
+  handleSubmit,
+  initialValues,
+  ledger,
+  onCancel,
+  submitting,
+  pristine,
+}) => {
+  const closeForm = useCallback(() => onCancel(), [onCancel]);
```

Comment:
Maybe not introduce `closeForm` and just use `onCancel`?
Or it's just a skeleton thing&